share/themes/palette_light_fusion.json

```diff
@@ -7,6 +7,7 @@
 "Button":        "#c9c9c9",
 "ButtonText":    "#333333",
 "BrightText":    "#000000",
+"Highlight":     "#b3d7ff",
```

Comment:
This PR is outdated. "Highlight" already exists in current version of this file.
I would test this PR with actual state of the styling .json file and verify whether it looks properly.